drain-service/drain_training_inferencing.py

```diff
@@ -326,6 +328,20 @@ def weighted_avg_and_std(values, weights):
 
                 training_start_ts_ns = time.time_ns()
 
+                drain_status_doc = {
```

Comment:
in `file_persistance.py`, timestamp field was `int(time.time() * 1000)`, and here it is `int(training_end_ts_ns / 1000000)`, is it expected?

Reply:
and `num_drain_templates` and `num_drain_clusters` are same thing, right?

Reply:
> in file_persistance.py, timestamp field was int(time.time() * 1000), and here it is int(training_end_ts_ns / 1000000), is it expected?

Yes this is intended. `int(time.time() * 1000)` will give us milliseconds and this timestamp is used for `drain_persist` status updates

For the training signal we are using `int(training_end_ts_ns / 1000000)` because we would like to **re-use** the exact **end** timestamp of the normal interval that will be used by training. This is for `training_signal` status updates